nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -269,8 +269,6 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - [Xastir](https://xastir.org/index.php/Main_Page) can now access AX.25 interfaces via the `libax25` package.
 
-- `tvbrowser-bin` was removed, and now `tvbrowser` is built from source.
```

Comment:
why not keep this?

Reply:
We might, but with the alias (below), it seems barely relevant.  Maybe @SuperSandro2000 can comment?

Certainly `tvbrowser` isn't a high-profile package.  I wouldn't mind keeping it, but I also wouldn't mind removing it.

Reply:
Drop in replace alias are not worthy release notes. If it isn't drop in, then the alias would be a throw.

Reply:
Not sure how to proceed here.  In the interest of consistency, I would still opt to drop the release notes entry unless release notes are common in similar cases.